pulumi/infra/model_plugin_deployer.py

```diff
@@ -0,0 +1,75 @@
+from typing import Optional
+
+from infra.bucket import Bucket
+from infra.config import GLOBAL_LAMBDA_ZIP_TAG, LOCAL_GRAPL, mg_alphas
+from infra.dynamodb import DynamoDB
+from infra.lambda_ import Lambda, LambdaExecutionRole, PythonLambdaArgs, code_path_for
+from infra.metric_forwarder import MetricForwarder
+from infra.network import Network
+from infra.secret import JWTSecret
+
+import pulumi
+
+
+class ModelPluginDeployer(pulumi.ComponentResource):
+    def __init__(
+        self,
+        network: Network,
+        db: DynamoDB,
+        secret: JWTSecret,
+        ux_bucket: Bucket,
+        plugins_bucket: Bucket,
+        forwarder: Optional[MetricForwarder] = None,
+        opts: Optional[pulumi.ResourceOptions] = None,
+    ) -> None:
+
+        name = "model-plugin-deployer"
+        super().__init__("grapl:ModelPluginDeployer", name, None, opts)
+
+        self.role = LambdaExecutionRole(name, opts=pulumi.ResourceOptions(parent=self))
+
+        self.function = Lambda(
+            name,
+            args=PythonLambdaArgs(
+                execution_role=self.role,
+                description=GLOBAL_LAMBDA_ZIP_TAG,
+                handler="lambdex_handler.handler",
+                code_path=code_path_for(name),
+                env={
+                    "GRAPL_LOG_LEVEL": "INFO",
+                    "MG_ALPHAS": mg_alphas(),
+                    "JWT_SECRET_ID": secret.secret.arn
+                    if not LOCAL_GRAPL
+                    else "JWT_SECRET_ID",
```

Comment:
I know black did this, not you, but I hate the formatting here - can we maybe just pull this out to a variable first?

Reply:
and tbh I Dont Care That Much